### PR TITLE
Documentation on stages and transitions ready for review (issue #690)

### PR DESCRIPTION
**Pull request by RainerKlute**

### CODE REVIEW

**Comment by RainerKlute** [comment]
Ready for review

File: plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile [hunk 1]
```diff
@@ -42,23 +42,223 @@ p=. Parallel regions
...
+If an "event":#events occurs, it may trigger a so-called _local reaction_ in the active state. It is executed completely inside the active state without leaving it, thus the word "local". The UML calls local reactions "internal transitions".
+
+Each local reaction consists of three parts, as shown by figure "&quot;Local reaction syntax&quot;":#fig_local-reaction-syntax.
+
+p(#fig_local-reaction-syntax). 
+!(standard-image)images/docu_grammar_030_state-localreaction.png(Local reaction syntax)!
+
+p=. Local reaction syntax
+
+* The _reaction trigger_. It specifies the conditions that have to be fulfilled for the local reaction to be executed. It consists of one or more "_event specifications_":#event-specifications and a "_guard condition_":#guard-condition. One of these, i.&nbsp;e. either event specification(s) or guard condition, may be omitted, but not both.
```
Comment (tkutz) [inline plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile]: In the preview, I see an unintended underscore at "reaction trigger_" and "_guard condition". Not sure why.

File: plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile [hunk 2]
```diff
@@ -42,23 +42,223 @@ p=. Parallel regions
...
+p=. Event specifications syntax overview
+
+h3(#guard-condition). Guard condition
+
+A _guard_ applies an additional condition for a "transition":#transition to fire or a "local reaction":#local-reactions to be executed. It is not sufficient for a specified event to occur, the guard condition must be fulfilled, too. It is written in square brackets following the "event specifications":#event-specifications. You can specify a guard condition without any event specification. In this case the behavior of the transition or the local reaction is as if the @always@ event had been specified. That is, the transition or local reaction are always executed, as long as the guard condition evaluates to _true_.
+
+The following examples assume the _x_ is an integer variable and _isOpen_ is an operation.
+
+|_. Transition specification: |_. Prerequisites for the transition to fire: |
+{vertical-align: top}. |{width: 37ex}  @ev [x == 3]@ | Event _ev_ occured and _x_ holds the value 3. |
```
Comment (tkutz) [inline plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile]: In the preview I can see "{width: 37ex}".

**Comment by tkutz** [comment]
review done, see comments

File: plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile (hunk 1 above)
Comment (RainerKlute) [inline plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile]: Seems to be a bug in the GitHub preview. The generated HTML (and the source code) are looking good.

File: plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile (hunk 2 above)
Comment (RainerKlute) [inline plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile]: Also a bug in the GitHub preview.

**Pull request merged**